solvebio/exporters.py

```diff
@@ -247,6 +261,98 @@ def write(self, filename):
             f.close()
 
 
+class FlatCSVExporter(CSVExporter):
```

Comment:
do you think we should just replace the existing CSV and XLS exporters with this code?